lib/buffer.js

```diff
@@ -351,7 +364,7 @@ Buffer.concat = function(list, length) {
     length = length >>> 0;
   }
 
-  var buffer = Buffer.allocUnsafe(length);
+  var buffer = allocate(length);
```

Comment:
This is Buffer.concat - it either sums up lengths of existing `Buffer` list (throws error if they're not `Buffer`s, and `length` of `Buffer` is always `>=0`) or performs `length >>> 0` which makes it also always `>= 0` (32-bit unsigned).